kfai_sql_chemistry/test/utils/models_for_testing/models.py

```diff
@@ -0,0 +1,25 @@
+from sqlalchemy import Column, ForeignKey, Integer
+from sqlalchemy.orm import declarative_base, relationship
+
+BaseModel = declarative_base()
```

Comment:
I would call this BaseDbModel (which I stole from the CU repo), so people don't confuse it for BaseModel in pydantic

Reply:
fair